pkg/appClone/AppCloneService.go

```diff
@@ -278,10 +278,13 @@ func (impl *AppCloneServiceImpl) CreateCiTemplate(oldAppId, newAppId int, userId
 		return nil, fmt.Errorf("no git for %d", newAppId)
 	}
 	dockerfileGitMaterial := 0
+	buildContextGitMaterial := 0
 	if len(gitMaterials) == 1 {
 		dockerfileGitMaterial = gitMaterials[0].Id
+		buildContextGitMaterial = gitMaterials[0].Id
 	} else {
 		refGitmaterial, err := impl.materialRepository.FindById(refCiConf.CiBuildConfig.GitMaterialId)
+		refBuildContextGitMaterial, err := impl.materialRepository.FindById(refCiConf.CiBuildConfig.BuildContextGitMaterialId)
```

Comment:
compare `refCiConf.CiBuildConfig.BuildContextGitMaterialId` with `refCiConf.CiBuildConfig.GitMaterialId` if same then we can avoid db call.